engineering/workflow.md

```diff
@@ -54,31 +55,153 @@ The team conducts a Sprint Planning meeting for 60 minutes at the beginning of e
+
+   Team members need ample time to reflect on their work, interactions and the effectiveness of process from the past iteration. This time will vary based on the size of the team, the duration of the retrospective and temperament of the team.
+
+   This duration of this step is usually between 5 - 15 minutes.
+
+   In some cases, the team is invited to pre-populate the board with input before the actual retrospective.
+
+1. _~~Everyone reads through (quietly to themselves) the~~ Thanks and Celebrations_
+
+    Ultimately, the team needs to find a way to celebrate each other. In some cases, this may be simply the team reading the cards themselves or the facilitator doing this.
+
+1. _Create shared understanding on Went Well, and To Improve columns._
+
+   During this time, the facilitator groups the cards into themes (in the form of hashtags), potentially merging similar cards.
+
+1. _Discussion amongst the team on the To Improve items._
+
+   For cards that are unclear, the facilitator should encourage the person that wrote the card to provide context. Team members can ask clarifying questions of each other.
+
+   **NOTE**: It is important to discourage "solutioning" at this stage.
+
+1. _Voting on the To Improve items._
+
+   The team can then vote on which To Improve items they think are the most important.
+
+   _For example: Each member gets 3 votes that they can distribute as they see fit, e.g., give all 3 votes to one item, 2 to one item and 1 to another, or 1 vote for 3 items._
+
+1. _Identify actions._
+
+   For the top-voted actions (limited to only 2 or 3), the team generates some concrete actions to try and resolve the issues.
+   These are captured in the Action Items column.
+
+1. _Close the meeting._
+
+  After the meeting, the facilitator is responsible for converting the Action Items into GitHub issues to be refined for the next sprint, and then clearing the retro board.
+
+#### When should the Retro Board be Populated
+
+Ideally, the team is expected to populate the retrospective board during the meeting. However, team members can pre-populate the board prior to the meeting if they have fleeting thoughts they'd like to capture throughout the sprint.
+
+#### How do Generated Actions move into and get committed to the Team's Next Sprint?
+
+The general rule is that actions are also work and should be prioritized like any other work.
+
+As such actions will enter the team's next sprint through the following process:
 
-- {guilabel}`Up Next` Items that are ready to be worked on.
-- {guilabel}`In progress` An item that a team member is currently working towards.
-- {guilabel}`Done` Items that are complete! These should be celebrated archived in the next Sprint Planning meeting.
+1. At the end of the retrospective, the facilitator should create issues representing the actions on the Team's iteration board, in the `Backlog` or `Refined` queue depending on the level of detail/information detailed in the action. 
```

Comment:
> The general rule is that actions are also work and should be refined and prioritized like any other work.

The leading paragraph quoted above combined with whats written here is sufficient for this heading I think.

By removing the rest of the headings content we avoid repeating process details on backlog refinement and iteration planning that I think we ought to assume is known and documented elsewhere when writing this section.

Reply:
Cool.